capstone/src/main/java/com/step902020/capstone/Event.java

```diff
@@ -134,4 +134,35 @@ public void addReview(Review review) {
     reviews.add(review);
   }
 
+  /**
+   * returns event information in a string format
+   * @return String
+   */
+  public String toString() {
+    return eventTitle + " " + datastoreId;
+  }
+
+  /**
+   * implement equality
+   * @param o object being compared to
+   * @return boolean
+   */
+  @Override
+  public boolean equals(Object o) {
```

Comment:
What is this used for?

Reply:
To override the object equals. When the events are returned from the database they are not the same objects anymore but they should be considered "equal" if their ids are equal.